docsrc/tutorials/getting_started.rst

```diff
@@ -201,6 +201,7 @@ to load in a deployment application. In order to load a TensorRT/TorchScript mod
 
     ...
 
+    trtorch.set_device(0) # Set CUDA device (in case of multi-gpu configuration). Defaults to 0
```

Comment:
I dont think that its hugely important for this to be in the getting started docs. Its more imporant to cover the semantics of selected device in the compile spec

Reply:
Added it for reference as to how to use it. Removed it anyways.